hms-app/src/pages/AllIdeas.tsx

```diff
@@ -325,8 +350,29 @@ function AllIdeas() {
 
                 <HackathonHeader hackathonData={hackathonData} />
 
+                <Stack align='flex-start' justify='flex-start' spacing='sm'>
+                  {showUserIdeas ? (
+                    <Title order={2} mt={50}>
+                      Your submission: {userIdeaList.length}
+                    </Title>
+                  ) : (
+                    <Title order={2} mt={50}>
+                      Ideas submitted: {relevantIdeaList.length}
+                    </Title>
+                  )}
```

Comment:
you could put the ternary inside the `<Title order...` tags to eliminate a little duplication